### PR TITLE
Bump Node.js versions

### PR DESCRIPTION
**Pull request by mattbaileyuk**
Bump the version of Node.js that Travis runs against to supported versions, and also bump the `engines` section to the new minimum version (to avoid all the current installation warnings, as it claims 4.4-only). As this removes versions from support, bump the major version to `3.0.0`

### CODE REVIEW

**Comment by mattbaileyuk** [comment]
It's probably because this repo has not been used in a long while so didn't catch the removal of travis-ci.org (free runs for open source projects) and the need to sync up with travis-ci.com. That will require someone with write access to set up, though; I wonder who has that...

**Review by bthomps — APPROVED** [review]

**Comment by mattbaileyuk** [comment]
Travis won't run on my branch on my fork, but having migrated the configuration over to travis-ci.com it _should_ run there on `master` after merging, and we can validate it's all good 🤞

**Pull request merged**
